Formula/qgis3.rb

```diff
@@ -374,41 +409,55 @@ def install
       -DWITH_INTERNAL_QWTPOLAR=FALSE
       -DQGIS_MACAPP_BUNDLE=0
       -DQGIS_MACAPP_INSTALL_DEV=FALSE
-      -DWITH_QSCIAPI=TRUE
+      -DWITH_QSCIAPI=FALSE
       -DWITH_STAGED_PLUGINS=TRUE
       -DWITH_CUSTOM_WIDGETS=TRUE
       -DWITH_ASTYLE=FALSE
-      -DUSE_CCACHE=OFF
+      -DWITH_GRASS=FALSE
     ]
 
+    # disable CCache
+    args << "-DUSE_CCACHE=OFF"
+
+    # By default bindings will be installed only to QGIS directory
+    # Someone might want to install it to python site-packages directory
+    # as otherwise user has to use PYTHONPATH environment variable to add
+    # args << "-DWITH_BINDINGS=ON"
+
     # python Configuration
     args << "-DPYTHON_EXECUTABLE='#{`python3 -c "import sys; print(sys.executable)"`.chomp}'"
     # args << "-DPYTHON_CUSTOM_FRAMEWORK='#{`python3 -c "import sys; print(sys.prefix)"`.chomp}'" # not used by the project
     # Disable future, because we've installed it in the virtualenv and will provide it at runtime.
     # args << "-DWITH_INTERNAL_FUTURE=FALSE" # not used by the project
-    #args << "-DPYTHON_INCLUDE_PATH=#{Formula["python"].opt_include}"
-    #args << "-DPYTHON_LIBRARY=/usr/local/Frameworks/Python.framework/Versions/#{py_ver}/Python"
-    #args << "-DPYTHON_SITE_PACKAGES_DIR=/usr/local/lib/python#{py_ver}/site-packages"
+    # args << "-DPYTHON_INCLUDE_PATH=#{HOMEBREW_PREFIX}/Frameworks/Python.framework/Versions/#{py_ver}/include/python#{py_ver}m"
+    # args << "-DPYTHON_LIBRARY=#{HOMEBREW_PREFIX}/Frameworks/Python.framework/Versions/#{py_ver}/Python"
+    # args << "-DPYTHON_LIBRARY=#{HOMEBREW_PREFIX}/Frameworks/Python.framework/Versions/#{py_ver}/lib/"
+    # args << "-DPYTHON_SITE_PACKAGES_DIR=#{HOMEBREW_PREFIX}/lib/python#{py_ver}/site-packages"
 
     # if using Homebrew's Python, make sure its components are always found first
     # see: https://github.com/Homebrew/homebrew/pull/28597
-    ENV["PYTHONHOME"] = python_prefix if brewed_python?
+    ENV["PYTHONHOME"] = python_prefix
 
-    # handle custom site-packages for qt keg-only modules and packages
-    ENV.prepend_path "PYTHONPATH", python_site_packages
-    ENV.append_path "PYTHONPATH", python_qt_site_packages
-    # ENV.append_path "PYTHONPATH", libexec/"vendor/lib/python#{py_ver}/site-packages"
-    ENV.prepend_path "PATH", libexec/'vendor/bin/'
-    # ENV.prepend_path "PATH", libexec/"python/bin"
+    # handle custom site-packages for keg-only modules and packages
+    ENV.append_path "PYTHONPATH", python_site_packages
```

Comment:
This seems to entirely defeat the purpose of vendoring our python resources. If we simply add the system site-packages into the path, before adding the vendored directory to the path, then we're relying on whatever packages and versions the users has installed. I don't see how that's better than vendoring the resources ourselves.

Reply:
I solved the pop-up window with
```
# handle custom site-packages for keg-only modules and packages
ENV.append_path "PYTHONPATH", libexec/"python/lib/python#{py_ver}/site-packages"
ENV.append_path "PYTHONPATH", lib/"python#{py_ver}/site-packages"
```
```
# set install's lib/python#{py_ver}/site-packages first, so app will work if unlinked
pypths = %W[
  #{opt_lib}/python#{py_ver}/site-packages
  #{opt_libexec}/python/lib/python#{py_ver}/site-packages
  #{pypth}
]
```